packs/elastic/pack.yaml

```diff
@@ -0,0 +1,10 @@
+---
+name : elastic
```

Comment:
I think "elasticsearch" is more appropriate name for this pack. See comment from the previous PR - https://github.com/StackStorm/st2contrib/pull/198#issuecomment-101238465